README.md

```diff
@@ -0,0 +1,51 @@
+# Wikispeech-Prerender
+
+Starting from scratch will 
+
+## REST
+
+Most REST calls are exist for debug and development reasons.
+As a user of this service, all you need is ```POST /api/wiki```
+
+### POST /api/wiki
+* consumerUrl: Wiki to be monitored for pre rendered.
+* initialLastRecentChangesLimitInMinutes: (default: 60) Number of hours of initial recent changes backlog to be processed.
+* mainPagePriority: (default: 10) Base priority of segments on Wiki main page.
```

Comment:
What are the priorities assigned to other queued items? I.e. does 10 make it highest priority, lowest or somewhere in between?